### ISSUE TITLE
Add page/ popup menu is not announced by the screen reader 

### ISSUE THREAD

**Issue by Chetna1307** [Chetna1307/Wordpress-#1]
UAT 2022-09
Windows 10
Chrome Version 98.0.4758.102 (Official Build) (64-bit)
NVDA 2021
Repro Steps
1. Open the URL https://wordpress.com/page/ankurguptamusic.wordpress.com/21
2.Press Tab key to navigate to the block editor after the title, press enter and start typing. Press tab key to come out of the edit box.
3.there is a popup menu on the screen, check if the screen reader announces the menu options using the keyboard or mouse.
Expected Result
The popup menu should be announced by the screen reader Actual Result
There is no indication of the popup menu on the screen WCAG Guideline
WCAG guide line
https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus
[add new page 1.webm](https://user-images.githubusercontent.com/102085971/191180565-a6ff1777-f730-4e2f-980f-d2c02be22745.webm)
